po/es.po

```diff
@@ -436,7 +441,8 @@ msgid ""
 "Copyright (C) 2003-%d Leandro A. F. Pereira. See COPYING for details.\n"
 "\n"
 msgstr ""
-"Copyright (C) 2003-%d Leandro A. F. Pereira. Vea COPYING para más detalles.\n"
+"Copyright (C) 2003-%d Leandro A. F. Pereira. Vea COPYING para obtener más"
+" detalles.\n"
```

Comment:
an extra line was added here please check if are enoguht space in the resulting window msg widget!

Reply:
Fixed.